tests/run.sh

```diff
@@ -20,12 +20,19 @@ else
     runwasi=""
 fi
 
+if [ $# -gt 1 ]; then
+    wasi_sdk="$2"
+else
+    wasi_sdk=""
+fi
```

Comment:
Can we just make this a required argument .. perhaps the first one, so the runtime can be the optional second arg?